Resources/Prototypes/Voice/disease_emotes.yml

```
@@ -3,6 +3,16 @@
   name: chat-emote-name-sneeze
   category: Vocal
   chatMessages: [ "chat-emote-msg-sneeze" ]
+  # WD EDIT START
+  whitelist:
+    components:
+    - Vocal
+  blacklist:
+    components:
+    - BorgChassis
+  chatTriggers:
+    - "chat-emote-trigger-sneeze-1"
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)